src/main/java/hudson/tasks/junit/StdioRetention.java

```diff
@@ -0,0 +1,15 @@
+package hudson.tasks.junit;
+
+public enum StdioRetention {
+
+    all,
+    failed,
+    none;
```

Comment:
normal code convention in Java is for enums to be in all uppercase letters.

Reply:
The intent was to permit lowercase values. Any idea if there's a way to enable case-insensitive parsing for enum types with `@DataBoundSetter` methods?

Reply:
Pushed a new commit that does this work manually; happy to change things if there's a cleaner approach.